File: Jour6/jour6.pas
program pariter;
var 
n : integer;
begin
    write('Donner un entier pour trouver sa paritee : ');
    read(n);
    //Attention aucun control sur la saisire de la variable
    if n mod 2 = 0 then
    begin
		writeln(n,' est un nombre pair.');
	end
	else
		writeln(n,' est un nombre impair.');
end.

File: Jour6/jour6.c
#include<stdio.h>

int main()
{
	int n;
	do
	{
		printf("Donner un entier pour trouver sa paritee : ");
		scanf("%d",&n);
		if(n != (int)n)
			printf("Erreur de saisie ::::: ressayer");
	}while(n != (int)n);
	
	if(n % 2 == 0)
		printf("%d est un nombre pair.",n);
	else
		printf("%d est un nombre impair.",n);
	
	return 0;
}